### PR TITLE
DIT-289: showing ruling information in the UI

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_nav
+
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+
+    <h1 class="heading-large">Not implemented yet!</h1>
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html]: I will fix this in the next PR

Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html]: Do we want to be doing "Not implemented yet" pull requests?

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                                <p id="bindingCommodityCodeValue">@{d.bindingCommodityCode}</p>
+                            </div>
+                        </div>
+
+                        <div class="grid-row">
+                            <div id="searches" class="column-one-third">
+                                <p>Searches</p>
+                            </div>
+                            <div class="column-two-thirds">
+                                <p id="searchesValue">???</p>
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html]: methodSearch is the field for this

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                        <div class="grid-row">
+                            <div id="sanitisedGoodDescription" class="column-one-third">
+                                <p>Good Description</p>
+                            </div>
+                            <div class="column-two-thirds">
+                                <p id="sanitisedGoodDescriptionValue">@{d.goodsDescription}</p>
+                            </div>
+                        </div>
+
+                        <div class="grid-row">
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html]: Can we comment this until we know what field it maps to

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+            <div class="column-three-quarters">
+
+                @if(c.decision.isDefined) {
+
+                    <div class="grid-row">
+                        <div id="ruling" class="column-one-third">
+                            <h4 class="heading-medium mt-0">Ruling</h4>
+                        </div>
+                        <div id="change-details" class="column-two-thirds">
+                            <p><a id="change" href="#">Change</a></p>
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html]: If we aren't displaying incomplete functionality we should probably comment this for now

**Pull request merged**